packages/app-project/src/screens/ProjectSubjectPage/ProjectSubjectPage.js

```diff
@@ -0,0 +1,127 @@
+import { Media } from '@zooniverse/react-components'
+import { Box, Grid } from 'grommet'
+import { Bookmark, Favorite, ShareOption, SubtractCircle } from 'grommet-icons'
+import { shape, string } from 'prop-types'
+import styled from 'styled-components'
+
+import ContentBox from '@shared/components/ContentBox'
+import StandardLayout from '@shared/components/StandardLayout'
+
+// based on the lib-classifier MaxWidth layout
+const ContainerGrid = styled(Grid)`
+  position: relative;
+  grid-gap: 1.875rem;
+  grid-template-areas: 'viewer talkData';
+  grid-template-columns: minmax(auto, 100rem) 25rem;
+  margin: auto;
+
+  @media screen and (max-width: 1024px) {
+    grid-gap: 1.25rem;
+    grid-template-areas:
+      'viewer'
+      'talkData';
+    grid-template-columns: 100%;
+    grid-template-rows: auto auto;
+    margin: 0;
+    width: 100%;
+  }
+`
+
+function ProjectSubjectPage({
+  appLoadingState,
```

Comment:
You've connected this prop from the store, but what will it be used for?

Reply:
I don't have a use for `appLoadingState` at the moment, so I'm going to remove it and the related connector component. I can easily add it back if needed.